libs/sql-schema-describer/tests/describers/postgres_describer_tests/cockroach_describer_tests.rs

```diff
@@ -359,3 +360,94 @@ fn cockroachdb_sequences_must_work(api: TestApi) {
+    let table = schema.table_walkers().next().unwrap();
+    let col = table.column("mysum").unwrap();
+    let value = col.default().unwrap().as_value().unwrap();
+    assert!(matches!(value, PrismaValue::Int(37)));
+}
+
+#[test_connector(tags(CockroachDb))]
+fn array_column_defaults(api: TestApi) {
+    let schema = r#"
+        CREATE TYPE "color" AS ENUM ('RED', 'GREEN', 'BLUE');
+
+        CREATE TABLE "defaults" (
+            text_empty TEXT[] NOT NULL DEFAULT '{}',
+            text TEXT[] NOT NULL DEFAULT '{ ''abc'' }',
+            text_c_escape TEXT[] NOT NULL DEFAULT E'{ \'abc\', \'def\' }',
+            colors COLOR[] NOT NULL DEFAULT '{ RED, GREEN }',
+            int_defaults INT4[] NOT NULL DEFAULT '{ 9, 12999, -4, 0, 1249849 }',
+            float_defaults DOUBLE PRECISION[] NOT NULL DEFAULT '{ 0, 9.12, 3.14, 0.1242, 124949.124949 }',
+            bool_defaults BOOLEAN[] NOT NULL DEFAULT '{ true, true, true, false }',
+            datetime_defaults TIMESTAMPTZ[] NOT NULL DEFAULT '{ "2022-09-01T08:00Z","2021-09-01T08:00Z"}'
+        );
+    "#;
+
+    api.raw_cmd(schema);
+    let schema = api.describe();
+    let table = schema.table_walkers().next().unwrap();
+
+    let assert_default = |colname: &str, expected_default: Vec<PrismaValue>| {
+        let col = table.column(colname).unwrap();
+        let value = dbg!(col.default().unwrap()).as_value().unwrap();
+        assert_eq!(value, &PrismaValue::List(expected_default));
+    };
+
+    assert_default("text_empty", vec![]);
+    assert_default("text", vec![PrismaValue::String("abc".to_owned())]);
+    assert_default("text_c_escape", vec!["abc".into(), "def".into()]);
+    assert_default(
+        "colors",
+        vec![
+            PrismaValue::Enum("RED".to_owned()),
+            PrismaValue::Enum("GREEN".to_owned()),
+        ],
+    );
+    assert_default(
+        "int_defaults",
+        vec![
+            PrismaValue::Int(9),
+            PrismaValue::Int(12999),
+            PrismaValue::Int(-4),
+            PrismaValue::Int(0),
+            PrismaValue::Int(1249849),
+        ],
+    );
+    assert_default(
+        "float_defaults",
+        vec![
+            PrismaValue::Float("0.0".parse().unwrap()),
+            PrismaValue::Float("9.12".parse().unwrap()),
+            PrismaValue::Float("3.14".parse().unwrap()),
+            PrismaValue::Float("0.1242".parse().unwrap()),
+            PrismaValue::Float("124949.124949".parse().unwrap()),
+        ],
+    );
+    assert_default(
+        "bool_defaults",
+        vec![
+            PrismaValue::Boolean(true),
+            PrismaValue::Boolean(true),
+            PrismaValue::Boolean(true),
+            PrismaValue::Boolean(false),
+        ],
+    );
+    // assert_default(
+    //     "datetime_defaults",
+    //     vec![
+    //         PrismaValue::Enum("2022-09-01 08:00:00+00".to_owned()),
+    //         PrismaValue::Enum("2021-09-01 08:00:00+00".to_owned()),
+    //     ],
+    // );
```

Comment:
Curious: why is this block commented out?

Reply:
it's a follow up PR